app/views/recipe_foods/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Recipe:</strong>
+  <%= @recipe_food.recipe_id %>
+</p>
+
+<p>
+  <strong>Food:</strong>
+  <%= @recipe_food.food_id %>
+</p>
+
+<p>
+  <strong>Quantity:</strong>
+  <%= @recipe_food.quantity %>
+</p>
+
+<%= link_to 'Edit', edit_recipe_food_path(@recipe_food) %> |
+<%= link_to 'Back', recipe_foods_path %>
```

Comment:
- [ ] `You can remove a recipe item only if you're the owner`, Please make sure you are preventing other users to get access to others' accounts or other's items or lists when he or she is not an owner.

Reply:
Fixed